Index: test_package.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- test_package.py	(revision b782fd9afdb91e3278222ea20137b5ca8ef48390)
+++ test_package.py	(date 1590259462916)
@@ -1,4 +1,4 @@
-from the_project import Module0
+from src.the_project import Module0
 
 
 def test_func0(fixture0: Module0):
Index: the_project_main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- the_project_main.py	(revision b782fd9afdb91e3278222ea20137b5ca8ef48390)
+++ the_project_main.py	(date 1590259769304)
@@ -1,13 +1,10 @@
-import sys
-from pathlib import Path
-
 from runtime_syspath import print_syspath
 
-PROJECT_DIR = Path(__file__).parent
-sys.path.append(str(PROJECT_DIR / 'src'))
-
 # pylint: disable=import-error
-from the_project import Module0  # noqa
+from the_project import Module0
+
+# PROJECT_DIR = Path(__file__).parent
+# sys.path.append(str(PROJECT_DIR / 'src'))
 
 # pylint: enable=import-error
 
Index: conftest.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- conftest.py	(revision b782fd9afdb91e3278222ea20137b5ca8ef48390)
+++ conftest.py	(date 1590259769301)
@@ -1,14 +1,11 @@
-import sys
-from pathlib import Path
-
 import pytest
 from runtime_syspath import print_syspath
 
-PROJECT_DIR = Path(__file__).parent
-sys.path.append(str(PROJECT_DIR / 'src'))
-
 # pylint: disable=import-error
-from the_project.module0 import Module0
+from src.the_project.module0 import Module0
+
+# PROJECT_DIR = Path(__file__).parent
+# sys.path.append(str(PROJECT_DIR / 'src'))
 
 # pylint: enable=import-error
 
