Index: test_package.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- test_package.py	(revision 001dc702d237b5bdf060782f21267eb48eeb6a48)
+++ test_package.py	(date 1590292836900)
@@ -1,4 +1,4 @@
-from the_project import Module0
+from src.the_project import Module0
 
 
 def test_func0(fixture0: Module0):
Index: .idea/the_project.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/the_project.iml	(revision 001dc702d237b5bdf060782f21267eb48eeb6a48)
+++ .idea/the_project.iml	(date 1590292836884)
@@ -1,9 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="PYTHON_MODULE" version="4">
   <component name="NewModuleRootManager">
-    <content url="file://$MODULE_DIR$">
-      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
-    </content>
+    <content url="file://$MODULE_DIR$" />
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
   </component>
Index: .idea/runConfigurations/Python_pytest_tests.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/runConfigurations/Python_pytest_tests.xml	(revision 001dc702d237b5bdf060782f21267eb48eeb6a48)
+++ .idea/runConfigurations/Python_pytest_tests.xml	(date 1590292836880)
@@ -2,9 +2,7 @@
   <configuration default="false" name="Python pytest tests" type="CompoundRunConfigurationType">
     <toRun name="test_all_the_tests_on_cli" type="PythonConfigurationType" />
     <toRun name="run_the_project_main" type="PythonConfigurationType" />
-    <toRun name="run_the_project_main_reusable_func" type="PythonConfigurationType" />
-    <toRun name="test_the_project_main_reusable_func" type="PythonConfigurationType" />
     <toRun name="test_package" type="PythonConfigurationType" />
     <method v="2" />
   </configuration>
-</component>
\ No newline at end of file
+</component>
Index: test_all_the_tests_on_cli.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- test_all_the_tests_on_cli.py	(revision 001dc702d237b5bdf060782f21267eb48eeb6a48)
+++ test_all_the_tests_on_cli.py	(date 1590292836897)
@@ -7,8 +7,8 @@
 TESTS: List[Path] = [
     Path('test_package.py'),
     Path('src/run_the_project_main.py'),
-    Path('test_the_project_main_reusable_func.py'),
-    Path('run_the_project_main_reusable_func.py')
+    # Path('test_the_project_main_reusable_func.py'),
+    # Path('run_the_project_main_reusable_func.py')
 ]
 
 
Index: src/run_the_project_main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/run_the_project_main.py	(revision 001dc702d237b5bdf060782f21267eb48eeb6a48)
+++ src/run_the_project_main.py	(date 1590292836888)
@@ -1,24 +1,10 @@
-import sys
-from pathlib import Path
-
 from runtime_syspath import print_syspath
 
-if __name__ != "__main__":
-    SRC_PATH = Path(__file__).parent
-    SRC_DIR_STR = str(SRC_PATH.absolute())
-    if SRC_DIR_STR not in sys.path:
-        sys.path.append(SRC_DIR_STR)
-        print_syspath(sort=False)
-else:
-    print_syspath(sort=False)
-
-# pylint: disable=import-error
+print_syspath(sort=False)
 
 # Why is Pycharm being dumb here? Answered with a Q on __main__ tests below...
 from the_project import Module0
 
-# pylint: disable=import-error
-
 print("Execute The Project.")
 
 
Index: conftest.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- conftest.py	(revision 001dc702d237b5bdf060782f21267eb48eeb6a48)
+++ conftest.py	(date 1590292836890)
@@ -1,22 +1,6 @@
-import sys
-from pathlib import Path
-
 import pytest
-from runtime_syspath import print_syspath
-
-print_syspath(sort=False)
-
-PROJECT_PATH = Path(__file__).parent.absolute()
-SRC_DIR_STR = str(PROJECT_PATH / 'src')
-if SRC_DIR_STR not in sys.path:
-    sys.path.append(SRC_DIR_STR)
-    print_syspath(sort=False)
-
-# pylint: disable=import-error
-from the_project import Module0
 
-
-# pylint: enable=import-error
+from src.the_project import Module0
 
 
 @pytest.fixture
Index: test_the_project_main_reusable_func.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- test_the_project_main_reusable_func.py	(revision 001dc702d237b5bdf060782f21267eb48eeb6a48)
+++ test_the_project_main_reusable_func.py	(date 1590292836903)
@@ -1,4 +1,4 @@
-from run_the_project_main import reusable_function
+from src.run_the_project_main import reusable_function
 
 
 def test_the_project_main_reusable_func():
Index: .idea/runConfigurations/Pytest_tests.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/runConfigurations/Pytest_tests.xml	(revision 001dc702d237b5bdf060782f21267eb48eeb6a48)
+++ .idea/runConfigurations/Pytest_tests.xml	(date 1590292836875)
@@ -1,7 +1,6 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="Pytest tests" type="CompoundRunConfigurationType">
     <toRun name="pytest for test_package.test_func0" type="tests" />
-    <toRun name="pytest for test_the_project_main_reusable_func.test_the_project_main_reusable_func" type="tests" />
     <method v="2" />
   </configuration>
-</component>
\ No newline at end of file
+</component>
Index: run_the_project_main_reusable_func.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- run_the_project_main_reusable_func.py	(revision 001dc702d237b5bdf060782f21267eb48eeb6a48)
+++ run_the_project_main_reusable_func.py	(date 1590292836893)
@@ -1,20 +1,4 @@
-import sys
-from pathlib import Path
-
-from runtime_syspath import print_syspath
-
-print_syspath(sort=False)
-
-PROJECT_PATH = Path(__file__).parent.absolute()
-SRC_DIR_STR = str(PROJECT_PATH / 'src')
-if SRC_DIR_STR not in sys.path:
-    sys.path.append(SRC_DIR_STR)
-    print_syspath(sort=False)
-
-# pylint: disable=import-error
-from run_the_project_main import reusable_function
-
-# pylint: enable=import-error
+from src.run_the_project_main import reusable_function
 
 if __name__ == "__main__":
     print(reusable_function())
